ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -125,3 +125,13 @@ def rfftfreq(n, d=1.0):
     N = n // 2 + 1
     results = ivy.arange(0, N, dtype=int)
     return results * val
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"1.24.3 and above": ("float16",)}, "numpy")
+def rfft2(a, s=None, axes=(-2, -1), norm=None):
+    a = ivy.array(a, "float64")
+    if norm is None:
+        norm = "backward"
+    s = a.shape if s is None else tuple(s)
+    return ivy.dft(a, axes=axes, inverse=False, onesided=True, dft_length=s, norm=norm)
```

Comment:
`ivy.dft` has an `axis` kwarg that expects the value to be an `int`, not a sequence of `ints`. You will have to change your implementation to either use some other suitable function from `ivy` or implement this function as a composition of `ivy` functions if you can't find a suitable function in `ivy`.